src/internal/agent/hooks/flux-helmrepo.go

```diff
@@ -37,6 +37,14 @@ func NewHelmRepositoryMutationHook(ctx context.Context, cluster *cluster.Cluster
 // mutateHelmRepo mutates the repository url to point to the repository URL defined in the ZarfState.
 func mutateHelmRepo(ctx context.Context, r *v1.AdmissionRequest, cluster *cluster.Cluster) (*operations.Result, error) {
 	l := logger.From(ctx)
+	var (
+		patches   []operations.PatchOperation
+		isPatched bool
+
+		isCreate = r.Operation == v1.Create
+		isUpdate = r.Operation == v1.Update
```

Comment:
Define these variables where they are first used rather than at the top of the function.

Reply:
I followed the pattern found in `src/internal/agent/hooks/flux-gitrepo.go`

```
	var (
		patches   []operations.PatchOperation
		isPatched bool

		isCreate = r.Operation == v1.Create
		isUpdate = r.Operation == v1.Update
	)
```
[`ref`](https://github.com/zarf-dev/zarf/blob/a21ed416fc2c2c1c80df97184a49433ee779b933/src/internal/agent/hooks/flux-gitrepo.go#L42-L48) but I can update and get that up tonight